util/test/sub_test

```diff
@@ -203,6 +203,11 @@ def LauncherTimeoutArgs(seconds):
 def ShellEscape(str):
     return re.sub(r'([\\!@#$%^&*()?\'"|<>[\]{} ])', r'\\\1', str)
 
+# Escape all special characters but leave spaces alone
+def ShellEscapeCommand(str):
+    return re.sub(r'([\\!@#$%^&*()?\'"|<>[\]{}])', r'\\\1', str)
```

Comment:
How did you come up with these characters?

Will you please use something other than str for the argument name? str() is a builtin function in python, so it's not really a good idea to use it (I realize `str` and `file` are used all over the place, but maybe we'll fix them all someday...).